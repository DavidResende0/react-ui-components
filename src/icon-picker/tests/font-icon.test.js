import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FontIcon from '../font-icon';

describe('FontIcon test', () => {
  let intialProps;
  beforeEach(() => {
    intialProps = {
      type: 'fa',
      icon: 'crosshairs',
    };
  });
  it('should render correctly in active state', () => {
    const wrapper = shallow(<FontIcon {...intialProps} isActive />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('should render correctly', () => {
    const wrapper = shallow(<FontIcon {...intialProps} />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
