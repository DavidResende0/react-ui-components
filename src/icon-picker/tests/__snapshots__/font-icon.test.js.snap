// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FontIcon test should render correctly 1`] = `
<i
  className="fa crosshairs iconlist-item "
  title=".fa.crosshairs"
/>
`;

exports[`FontIcon test should render correctly in active state 1`] = `
<i
  className="fa crosshairs iconlist-item active"
  title=".fa.crosshairs"
/>
`;
